src/pages/Onboarding/Onboarding.vue

```diff
@@ -5,93 +5,75 @@
-          </div>
-          <div class="content">
-            <h1>Create</h1>
-            <p>
-              First, you create a new card and we assign you a key.
-              This number (key) will be used to identify your transactions.
-            </p>
-          </div>
-        </section>
-        <section id="name-image">
+        <section id="app-browser">
           <div class="screenshot-phone">
             <div class="image">
-              <img src="./../../assets/img/onboarding/identity-card3.svg">
+              <div class="icons">
+                <img src="@/assets/img/onboarding/app-icon1.svg">
+                <img src="@/assets/img/onboarding/app-icon2.svg">
+                <img src="@/assets/img/onboarding/app-icon3.svg">
+                <img src="@/assets/img/onboarding/app-icon4.svg">
+                <img src="@/assets/img/onboarding/app-icon5.svg">
+              </div>
             </div>
           </div>
           <div class="content">
-            <h1>Name</h1>
+            <h1>The Aepp Ecosystem</h1>
             <p>
-              Next, you select a profile image and assign a name to your card.
-              This will help you to easily distinguish between your cards (identities).
+              Explore the aeternity aepp ecosystem and use them with a desired account.
             </p>
           </div>
         </section>
-        <section id="app-browser">
-          <div class="screenshot-phone">
+        <section id="active-account">
+          <div class="screenshot-phone2">
             <div class="image">
-              <div class="icons">
-                <img src="./../../assets/img/onboarding/app-icon1.svg">
-                <img src="./../../assets/img/onboarding/app-icon2.svg">
-                <img src="./../../assets/img/onboarding/app-icon3.svg">
-                <img src="./../../assets/img/onboarding/app-icon4.svg">
-                <img src="./../../assets/img/onboarding/app-icon5.svg">
-              </div>
+              <img src="@/assets/img/onboarding/dummy-list-view.svg">
+              <img src="@/assets/img/onboarding/mini-card-collapsed.svg">
             </div>
           </div>
           <div class="content">
-            <h1>Browse</h1>
+            <h1>Your active account</h1>
             <p>
-              Take a look through the preloaded apps available or add other apps of your choice to the interface.
+              Aepps automatically use the active account, displayed in the bottom. The grid icon takes you back to the browser.
             </p>
           </div>
         </section>
         <section id="end">
-          <div class="content">
-            <div class="screenshot-phone2">
+          <div class="screenshot-phone">
               <div class="image">
-                <img src="./../../assets/img/onboarding/dummy-list-view.svg">
-                <img src="./../../assets/img/onboarding/mini-card.svg">
+                <img src="@/assets/img/onboarding/card1.svg">
```

Comment:
On screen about AirGap used asset with collapsed secure account instead of usual one.